hazelcast/src/main/java/com/hazelcast/internal/eviction/EvictionPolicyType.java

```diff
@@ -29,8 +29,17 @@
     /**
```

Comment:
Please add a few lines of explanation for the feeble minded people like me what the purpose is. I guess random picks random items and none doesn't evict at all.